packages/serverless/README.md

```diff
@@ -41,3 +41,14 @@ exports.handler = Sentry.AWSLambda.wrapHandler((event, context, callback) => {
   throw new Error('oh, hello there!');
 });
 ```
+
+If you also want to trace performance of all the incoming requests and also outgoing AWS service requests, just set the `tracesSampleRate` option. 
+
+```javascript
+import * as Sentry from '@sentry/serverless/awslambda';
```

Comment:
I believe it should either say:

```js
import * as Sentry from '@sentry/serverless/awslambda';
Sentry.init(...);
```

or

```js
import * as Sentry from '@sentry/serverless';
Sentry.AWSLambda.init(...);
```

What about tree-shaking? If one is using AWSLambda, will it strip the GCPFunction code down?

Reply:
`import * as Sentry from '@sentry/serverless/awslambda'` was a typo. It was intended to be:

```javascript
import * as Sentry from '@sentry/serverless';
Sentry.AWSLambda.init(...);
```

As for tree-shaking, I need to check it.